tests/lib/prepare-project.sh

```diff
@@ -119,18 +121,15 @@ if [[ "$SPREAD_SYSTEM" == ubuntu-14.04-* ]]; then
     quiet apt-get install -y --force-yes apparmor libapparmor1 seccomp libseccomp2 systemd cgroup-lite util-linux
 fi
 
-quiet apt-get purge -y snapd
+distro_purge_package snapd
```

Comment:
Well, the package name here serves really like an "interface" which has different implementations. Having something like `install_package debian:snapd` means we have to touch a lot of test cases when we add a new distribution and this doesn't even respect different versions of those distributions yet. 

Expressing a general name like "netcat" which then is resolved to the correct set of packages needed on a given distributions takes away those extra maintenance steps we have to do for every test case and limits any changes to a single file which can abstract anything by distribution and by their versions.